client/components/tinymce/plugins/contact-form/dialog/field-edit-button.jsx

```diff
@@ -0,0 +1,55 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes } from 'react';
+import PureRenderMixin from 'react-pure-render/mixin';
+import classNames from 'classnames';
+
+/**
+ * Internal dependencies
+ */
+import Button from 'components/button';
+import Gridicon from 'components/gridicon';
+import Popover from 'components/popover';
+
+export default React.createClass( {
+	displayName: 'ContactFormDialogFieldEditButton',
+
+	mixins: [ PureRenderMixin ],
+
+	propTypes: {
+		expanded: PropTypes.bool.isRequired
+	},
+
+	getInitialState: function() {
+		return {
+			showTooltip: false
+		};
+	},
+
+	render() {
+		const classes = classNames("editor-contact-form-modal-field__edit", {
```

Comment:
From the eslint warning, we're expecting a `dialog` prefix. cc @mtias should we ignore this one, or use something like `dialog-editor-contact-form-field__edit`?